Source/Dafny/AST/DafnyAst.cs

```diff
@@ -426,6 +426,16 @@ public class Attributes {
       return attrs.AsEnumerable().Any(aa => aa.Name == nm);
     }
 
+    /// <summary>
+    /// Returns first occurrence of an attribute named "nm", or "null" if there is no such
```

Comment:
I made the change. But having done it, I actually prefer the quotes, because they make the comment more readable in a text buffer. I suppose Rider may add code formatting in the future.